photo_points/Repositories/AdminReviewRepository.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+using photo_points.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace photo_points.Repositories
+{
+    public class AdminReviewRepository : IAdminReviewRepository
+    {
+        private readonly PhotoDataContext _dbcontext;
+
+        public List<Capture> repo = new List<Capture> {
```

Comment:
This fake data can be removed now that we are populating the db with other fake data, but you should also make sure that all of the same things are entered in the db somehow